### ISSUE TITLE
Obj_loss in validation set loss drops first and then rises，what parameters can I modify to solve this problem？

### ISSUE THREAD

**Issue by 15050188022** [ultralytics/yolov5#7327]
### Search before asking

- [x] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Obj_loss in validation set loss drops first and then rises，and the mAP is unsatisfactory，
what parameters can I modify to solve this problem？
![image](https://user-images.githubusercontent.com/30464411/162130745-7ce5fa22-2f28-4098-b473-807b2c0d1f8f.png)

Thank you！

**Comment by glenn-jocher** [comment]
@15050188022 your training is overfitting on objectness. You can try modifying obj hyps:
https://github.com/ultralytics/yolov5/blob/0ca85ed65f124871fa7686dcf0efbd8dc9699856/data/hyps/hyp.scratch-low.yaml#L16-L17

and using a higher augmentation hyp file like hyp.scratch-high.yaml

**Comment by callmesora** [comment]
> @15050188022 your training is overfitting on objectness. You can try modifying obj hyps:
> 
> https://github.com/ultralytics/yolov5/blob/0ca85ed65f124871fa7686dcf0efbd8dc9699856/data/hyps/hyp.scratch-low.yaml#L16-L17
> 
> and using a higher augmentation hyp file like hyp.scratch-high.yaml

How much should we reduce these values? Just to have an idea of how much, would 0.8 be too much of a decrease?

**Comment by 18789447455** [comment]
> ### 询问前先搜索
> * [x] 我搜索了YOLOv5的[问题](https://github.com/ultralytics/yolov5/issues)和[讨论](https://github.com/ultralytics/yolov5/discussions)，没有发现类似的问题。
> 
> ### 问题
> 验证集loss中的obj_loss先下降后上升，mAP不理想， 修改什么参数可以解决这个问题？ ![图片](https://user-images.githubusercontent.com/30464411/162130745-7ce5fa22-2f28-4098-b473-807b2c0d1f8f.png)
> 
> 谢谢！
> 
> ### 额外的
> _没有反应_



> ### 了解前先搜索
> * [x] 我搜索了 YOLOv5 的[问题](https://github.com/ultralytics/yolov5/issues)和[讨论](https://github.com/ultralytics/yolov5/discussions)，没有发现类似的问题。
> 
> ### 问题
> 验证集loss中的obj_loss先下降后修改，mAP不理想， 什么参数可以解决这个问题？ ![图片](https://user-images.githubusercontent.com/30464411/162130745-7ce5fa22-2f28-4098-b473-807b2c0d1f8f.png)
> 
> 谢谢！
> 
> ### 额外的
> _没有反应_



> ### 询问前先搜索
> * [x] 我搜索了YOLOv5的[问题](https://github.com/ultralytics/yolov5/issues)和[讨论](https://github.com/ultralytics/yolov5/discussions)，没有发现类似的问题。
> 
> ### 问题
> 验证集loss中的obj_loss先下降后上升，mAP不理想， 修改什么参数可以解决这个问题？ ![图片](https://user-images.githubusercontent.com/30464411/162130745-7ce5fa22-2f28-4098-b473-807b2c0d1f8f.png)
> 
> 谢谢！
> 
> ### 额外的
> _没有反应_

请问你解决这个问题了吗

**Comment by ckyrkou** [comment]
Any update on how to compat this behaviour? In previous versions of YOLO the objectness score and non-objectness where separate which gave a way to compromise between the two. Now where everything is computed in the same BCE it is a bit confusing. What should be higher obj or obj_pw??

**Comment by glenn-jocher** [comment]
@ckyrkou the separation of objectness score and non-objectness was a unique aspect in previous YOLO versions. In YOLOv5, the trade-off is achieved through the balance of obj and obj_pw hyperparameters. A higher obj hyperparameter gives more weight to objectness loss, while a higher obj_pw hyperparameter emphasizes the no-objectness loss. For your case, adjusting both may be necessary to find the correct balance. Keep experimenting and monitoring the validation results to determine the optimal combination. Would you like further assistance with this?
